HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -189,6 +190,57 @@ def self.apply_header(hpxml, epw_file)
     end
   end
 
+  def self.apply_emissions_scenarios(hpxml)
+    hpxml.header.emissions_scenarios.each do |scenario|
+      default_units = HPXML::EmissionsScenario::UnitsLbPerMBtu
+      if scenario.emissions_type.downcase == 'co2'
```

Comment:
@shorowit I had accidentally entered `C02` (with a zero instead of an O), and so this didn't default fossil fuels. Should we protect against that?

Reply:
OS-HPXML will provide a warning if a value is not provided and it's not automatically defaulted. I'm not sure what else to do here -- I want to allow for flexibility of many different types of emissions that users could want (TDV, source energy, PM2.5, etc etc) and we can't possibly have default values for every possible emissions type. If you have an idea on how to improve this, I'm all ears.

Reply:
I understand. I suppose my only idea would be to restrict the emissions type to be in a given list, but make the list comprehensive.

Reply:
I can't even big to imagine what a comprehensive list would look like. There are dozens and dozens of possibilities just based on what EPA reports.